install.sh

```diff
@@ -262,6 +262,60 @@ outdated_glibc() {
   version_lt "$glibc_version" "$REQUIRED_GLIBC_VERSION"
 }
 
+usage() {
+  cat <<EOS
+Homebrew Installer
+Usage: /bin/bash install.sh [--help] [--brew-git-remote URL] [--core-git-remote URL]
+
+This script installs Homebrew, the missing package manager for macOS (or Linux).
+
+Options:
+  -b, --brew-git-remote=URL   Use this URL as the Homebrew/brew git remote.
+                              Default: ${HOMEBREW_BREW_DEFAULT_GIT_REMOTE}.
+
+  -c, --core-git-remote=URL   Use this URL as the Homebrew/homebrew-core git remote.
+                              Default: ${HOMEBREW_CORE_DEFAULT_GIT_REMOTE}.
+
+  -h, --help                  Show this help message and exit.
+EOS
+  exit "${1:-0}"
+}
+
+# Parse remote URLs of Homebrew repositories from command line arguments if presented.
+parse_remote_urls_from_arguments() {
+  while [[ "$#" -gt 0 ]]; do
+    opt="$1"
+    shift  # expose the next argument
+    case "$opt" in
+      --) break 2 ;;
```

Comment:
Two questions:
1. In what situations do we need this? This script doesn't pass control to any other script.
2. `break` exits `for`, `while`, and `until` loops. Why do you break two levels here?